src/context_store/storage/prisma.py

```
@@ -0,0 +1,163 @@
+
+        for file_path in files:
+            version = file_path.name
+            if version not in applied:
+                await self._apply_migration(file_path)
+                logger.info("Applied migration via Prisma: %s", version)
+
+    async def _ensure_system_migration(self, file_path: Path) -> None:
+        try:
+            await self._client.query_raw("SELECT 1 FROM schema_migrations LIMIT 1")
+            return
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)